sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/login/AADLoginIT.java

```diff
@@ -0,0 +1,128 @@
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverWin32);
+        } else if (Pattern.matches("mac.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverMac);
+        }
+    }
+
+//    @Test
+    public void loginTest() {
+        this.runTest(app -> {
+            WebDriver driver = new ChromeDriver();
+            WebDriverWait wait = new WebDriverWait(driver, 10);
+            driver.get(app.root() + "api/group1");
+            wait.until(presenceOfElementLocated(By.name("loginfmt")))
+                .sendKeys(System.getenv(AAD_USER_NAME_1) + Keys.ENTER);
+            wait.until(presenceOfElementLocated(By.name("passwd")))
+                .sendKeys(System.getenv(AAD_USER_PASSWORD_1));
+
+            try {
+                Thread.sleep(3000);
+                driver.findElement(By.cssSelector("input[type='submit']")).click();
+                Thread.sleep(3000);
+                driver.findElement(By.cssSelector("input[type='submit']")).click();
+                Thread.sleep(3000);
+                Assert.assertEquals("group1", driver.findElement(By.tagName("body")).getText());
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e);
+            } finally {
+                driver.quit();
+            }
+        });
+    }
+
+    public void runTest(Consumer<AppRunner> command) {
+        try (AppRunner app = new AppRunner(AADLoginIT.DumbApp.class)) {
+            app.property("spring.security.oauth2.client.registration.azure.tenant-id",
+                System.getenv("AAD_TENANT_ID_1"));
+            app.property("spring.security.oauth2.client.registration.azure.client-id",
+                System.getenv(AAD_MULTI_TENANT_CLIENT_ID));
+            app.property("spring.security.oauth2.client.registration.azure.client-secret",
+                System.getenv(AAD_MULTI_TENANT_CLIENT_SECRET));
+
+            app.property("azure.activedirectory.tenant-id", System.getenv(AAD_TENANT_ID_1));
+            app.property("azure.activedirectory.client-id", System.getenv(AAD_MULTI_TENANT_CLIENT_ID));
+            app.property("azure.activedirectory.client-secret", System.getenv(AAD_MULTI_TENANT_CLIENT_SECRET));
+            app.property("azure.activedirectory.user-group.allowed-groups", "group1");
+
+            app.start();
+            command.accept(app);
+        }
+    }
+
+    @EnableGlobalMethodSecurity(securedEnabled = true, prePostEnabled = true)
+    @SpringBootApplication
+    @RestController
+    public static class DumbApp extends WebSecurityConfigurerAdapter {
+
+        @Autowired
+        private OAuth2UserService<OidcUserRequest, OidcUser> oidcUserService;
+
+        @PreAuthorize("hasRole('ROLE_group1')")
+        @GetMapping(value = "/api/group1")
+        public ResponseEntity<String> getRoleGroup1() {
+            return new ResponseEntity<>("group1", HttpStatus.OK);
+        }
```

Comment:
Now we tested `/api/group`. Could you please add the following 2 apis and test them?

1. `/api/home`
2. `api/group2`